addons/phone_validation/models/phone_blacklist.py

```diff
@@ -37,7 +37,7 @@ def create(self, values):
             try:
                 sanitized_value = self.env.user._phone_format(number=value['number'], raise_exception=True)
             except UserError as err:
-                raise UserError(str(err) + _(" Please correct the number and try again.")) from err
+                raise UserError(_("%(error)s Please correct the number and try again.", error=str(err))) from err
```

Comment:
```suggestion
                raise UserError(_("%(error)s Please correct the number and try again.", error=err)) from err
```
the conversion is done in the `%s`